packages/client/src/pages/SignIn/index.tsx

```diff
@@ -1,3 +1,83 @@
+import { Link } from 'react-router-dom';
+import { useActions } from '../../hooks/useActions';
+import { useTypeSelector } from '../../hooks/useTypeSelector';
+import { RouterList } from '../../router/routerList';
+import {
+  useLazyCheckAuthUserQuery,
+  useLogoutMutation,
+  useSignInMutation,
+} from '../../store/auth/api';
+import { useLazyFetchUserDataQuery } from '../../store/user/api';
+import type { RequestSignIn } from '../../store/auth/type';
+import type { RequestUserData } from '../../store/user/type';
+
 export default function SignIn() {
-  return <>SignIn</>;
+  const { isAuth } = useTypeSelector(state => state.auth);
+  const { userData } = useTypeSelector(state => state.user);
+
+  const [fetchAuthUser] = useLazyCheckAuthUserQuery();
+  const [fetchUserData] = useLazyFetchUserDataQuery();
+  const [fetchLogout] = useLogoutMutation();
+  const [fetchSignIn] = useSignInMutation();
+
+  const { setIsAuth, resetUserState, setUserData } = useActions();
+
+  const handleCheckIsAuth = () => {
+    fetchAuthUser(null).then(res => {
+      setIsAuth(res.isSuccess);
+    });
+  };
+
+  const handleSignIn = () => {
+    if (!isAuth) {
+      const params: RequestSignIn = {
+        login: 'Xxxxx',
+        password: '123XXXxxx',
+      };
+
+      fetchSignIn(params).then(() => {
+        setIsAuth(true);
+
+        if (userData?.id) {
+          const params: RequestUserData = { id: userData.id };
+
+          fetchUserData(params).then(res => {
```

Comment:
```suggestion
          fetchUserData(params).then(user => {
```
Хорошей практикой считается сразу указывать, какие именно данные находятся в `res`, чтобы не выводить это из кода ниже. В данном случае это `user`